packages/mom5/package.py

```diff
@@ -0,0 +1,320 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# Copyright 2023 ACCESS-NRI
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import install, join_path, mkdirp
+
+# https://spack.readthedocs.io/en/latest/build_systems/makefilepackage.html
+class Mom5(MakefilePackage):
+    """MOM is a numerical ocean model based on the hydrostatic primitive equations."""
+
+    homepage = "https://www.access-nri.org.au"
+    git = "https://github.com/ACCESS-NRI/mom5.git"
+
+    maintainers = ["harshula"]
+
+    version("master", branch="master")
+
+    # Depend on virtual package "mpi".
+    depends_on("mpi")
+    depends_on("oasis3-mct")
+    depends_on("datetime-fortran")
+    depends_on("netcdf-fortran@4.5.2:")
+    depends_on("netcdf-c@4.7.4:")
+    # TODO: For initial verification we are going to use static pio.
+    #       Eventually we plan to move to shared pio
+    # ~shared requires: https://github.com/spack/spack/pull/34837
+    depends_on("parallelio~pnetcdf~timing~shared")
+    depends_on("libaccessom2")
+
+    phases = ["edit", "build", "install"]
+
+    _platform = "spack"
+    _mom_type = "ACCESS-OM"
+
+    def edit(self, spec, prefix):
+
+        srcdir = self.stage.source_path
+        makeinc_path = join_path(srcdir, "bin", "mkmf.template.spack")
+        config = {}
+
+        incs = join_path((spec["oasis3-mct"].headers).cpp_flags, "psmile.MPI1 ")
```

Comment:
As expressed in a previous comment, I don't think trailing spaces in strings is a good approach and prefer what you have done in other packages and below in the loop:
```suggestion
        incs = join_path((spec["oasis3-mct"].headers).cpp_flags, "psmile.MPI1") + " "
```

Reply:
Yes, that is clearer.